sentry/src/main/java/io/sentry/ExternalOptions.java

```diff
@@ -25,6 +25,7 @@ public final class ExternalOptions {
   private @Nullable Boolean enableDeduplication;
   private @Nullable Double tracesSampleRate;
   private @Nullable Double profilesSampleRate;
+  private @Nullable Double logsSampleRate;
```

Comment:
to be removed as we don't sample logs

Reply:
Removed in https://github.com/getsentry/sentry-java/pull/4389